packages/stack/blockchain/src/index.js

```diff
@@ -109,6 +110,13 @@ export default class Blockchain {
     });
     this.blockchainApi.registerAPIs("ethereum");
     this.blockchainApi.registerRequests("ethereum");
+
+    if (this.blockchainConfig.enabled && this.blockchainConfig.client === "geth") {
+      warnIfPackageNotDefinedLocally("embark-geth", this.embark.logger.info.bind(this.embark.logger));
+    }
+    if (this.blockchainConfig.enabled && this.blockchainConfig.client === "parity") {
+      warnIfPackageNotDefinedLocally("embark-parity", this.embark.logger.info.bind(this.embark.logger));
```

Comment:
Whatever, not gonna repeat on each

Reply:
Also, feels weird that `stack/blockchain` warns about that. Wouldn't it be possible that a user creates his own Geth plugin? Then he'd get warning for no reason.
I know it's a scenario that will probably never happen, but it shows that it's weird that the parent knows the child plugins.
Maybe just a fix would be to show this warning only if no plugin is registered?